rebasehelper/spec_hooks/pypi_url_fix.py

```diff
@@ -36,6 +36,9 @@ class PyPIURLFixHook(BaseSpecHook):
     SOURCES_URL_TRANSFORMATIONS = [
         (r'https?://pypi(\.python)?\.org/', 'https://files.pythonhosted.org/'),
     ]
+    URL_TRANSFORMATIONS = [
+        (r'https?://pypi(\.python)?\.org/', 'https://pypi.org/'),
```

Comment:
I think the actual change is from `https://pypi.python.org/pypi/<NAME>` to `https://pypi.org/project/<NAME>`.